module/actor/ghoul-actor-sheet.js

```diff
@@ -0,0 +1,186 @@
+      potence: [],
+      presence: [],
+      protean: [],
+      sorcery: [],
+      oblivion: [],
+      rituals: [],
+      ceremonies: [],
+      alchemy: []
+    }
+
+    // Iterate through items, allocating to containers
+    for (const i of sheetData.items) {
+      if (i.type === 'power') {
+        // Append to disciplines.
+        if (i.data.discipline !== undefined) {
+          disciplines[i.data.discipline].push(i)
+          if (!this.actor.data.data.disciplines[i.data.discipline].visible) {
+            this.actor.update({ [`data.disciplines.${i.data.discipline}.visible`]: true })
+          }
+        }
+      }
+    }
+
+    // Assign and return
+    actorData.disciplines_list = disciplines
+  }
+
+  /* -------------------------------------------- */
+
+  /** @override */
+  activateListeners (html) {
+    super.activateListeners(html)
+
+    // Everything below here is only needed if the sheet is editable
+    if (!this.options.editable) return
+
+     // Make Discipline visible
+    html.find('.discipline-create').click(this._onShowDiscipline.bind(this))
+
+    // Make Discipline hidden
+    html.find('.discipline-delete').click(ev => {
+      const data = $(ev.currentTarget)[0].dataset
+      this.actor.update({ [`data.disciplines.${data.discipline}.visible`]: false })
+    })
+
+    // Rollable Vampire/Ghouls powers
+    html.find('.power-rollable').click(this._onVampireRoll.bind(this))
+  }
+
+  /**
+     * Handle making a discipline visible
+     * @param {Event} event   The originating click event
+     * @private
+     */
+  _onShowDiscipline (event) {
+    event.preventDefault()
+    let options = ''
+    for (const [key, value] of Object.entries(this.actor.data.data.disciplines)) {
+      options = options.concat(`<option value="${key}">${game.i18n.localize(value.name)}</option>`)
+    }
+
+    const template = `
+      <form>
+          <div class="form-group">
+              <label>${game.i18n.localize('VTM5E.SelectDiscipline')}</label>
+              <select id="disciplineSelect">${options}</select>
+          </div>
+      </form>`
+
+    let buttons = {}
+    buttons = {
+      draw: {
+        icon: '<i class="fas fa-check"></i>',
+        label: game.i18n.localize('VTM5E.Add'),
+        callback: async (html) => {
+          const discipline = html.find('#disciplineSelect')[0].value
+          this.actor.update({ [`data.disciplines.${discipline}.visible`]: true })
+        }
+      },
+      cancel: {
+        icon: '<i class="fas fa-times"></i>',
+        label: game.i18n.localize('VTM5E.Cancel')
+      }
+    }
+
+    new Dialog({
+      title: game.i18n.localize('VTM5E.AddDiscipline'),
+      content: template,
+      buttons: buttons,
+      default: 'draw'
+    }).render(true)
+  }
+
+  
+  _onVampireRoll (event) {
+    event.preventDefault()
+    const element = event.currentTarget
+    const dataset = element.dataset
+    const item = this.actor.items.get(dataset.id)
+    let disciplineValue = 1
```

Comment:
Important stuff : Ghouls only has 1 die for disciplines, even if they can learn powers.